Makefile

```diff
@@ -181,6 +186,13 @@ ifdef IMAGE_IMPORT_CMD
 	$(IMAGE_IMPORT_CMD) ${IMAGE_FULL_PATH}
 endif
 
+.PHONY: image-e2e
+image-e2e: ## Build docker image with the manager.
+	$(CONTAINER_TOOL) build -t ${IMAGE_FULL_PATH} -f tests/e2e/coverage/Dockerfile .
+ifdef IMAGE_IMPORT_CMD
+	$(IMAGE_IMPORT_CMD) ${IMAGE_FULL_PATH}
+endif
```

Comment:
may be able to consolidate this and `make image` into one by using an environment variable for the Dockerfile